app/models/user.rb

```diff
@@ -46,10 +46,10 @@ module Frequency
   # validates_attachment_content_type :photo_file_name, :content_type => %w(image/jpeg image/jpg image/png)
 
   has_many :images, foreign_key: :uid
-  has_many :node, foreign_key: 'uid'
+  has_many :node, foreign_key: 'uid', dependent: :destroy
```

Comment:
Confirmed that this will, when the user is destroyed, delete all the nodes and revisions:

https://guides.rubyonrails.org/association_basics.html#why-associations-questionmark